pages/about.html

```diff
@@ -4,9 +4,100 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="styles/style.css" rel=Stylesheet>
 </head>
-<body>
+<body> 
+  <header class="about-header">
+    <nav class="ss-bg">
+      <ul id="menu" class="about-menu">
+        <li><a href="http://127.0.0.1:5502/pages/index.html"><h3>Home</h3></a></li>
+        <li><a href="http://127.0.0.1:5502/pages/about.html"><h3>About</h3></a></li>
+        <li><img id="About-me-logo" alt="logo" src="../assets/ABOUT+ME+LOGO.png"></li>
+        <li><a href="http://127.0.0.1:5502/pages/portfolio.html"><h3>Portfolio</h3></a></li>
+        <li><a href="https://www.linkedin.com/in/wanjun-lan/"><h3>Contact</h3></a></li>
+      </ul>
+    </nav>
+  </header> 
+  <div id="div-grid">
```

Comment:
It seems there are a lot of divs being used on your pages. Instead, please consider using semantic tags like `<section>` or `<article>`. Screen readers tell blind users when a new section or a new article starts. It does not do that with `<div>`. Users will not know that they are on a new section of your page.